libraries/anaconda35/Dockerfile

```diff
@@ -1,19 +1,19 @@
 ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
 ENV PATH /opt/conda/bin:$PATH
 ENV ANACONDA_ENV=/home/algo/anaconda_environment
+ENV PYTHONDONTWRITEBYTECODE=true
```

Comment:
Overall, I'm hoping this is fine but it does make me a bit nervous. I'm guessing load time might be a bit higher for these algorithms since imports will take a bit more time, but that is outweighed by the speed of downloading a smaller image?

Reply:
honestly we can remove this. This was recommended from a "how to reduce anaconda dockerfile bloat"; but it was a very minor difference.